obj/objects.cpp

```diff
@@ -55,6 +55,10 @@ bool Sphere::hitObject(const core::Ray &ray, float freq,
   return true;
 }
 
+bool Sphere::isVecInside(const core::Vec3 &vec) const {
+  return (vec - origin_).magnitude() < radius_;
```

Comment:
Why does being equally at the radius does not count?

Reply:
because when ray is situated exactly at the radius, we dont hit the object, so to be consistent I made this case not inside the sphere  ;)

Reply:
I fail to see why consistency here is desired? These seems to be like completely independent pieces.

When ray is below the radius by kAccuracy it also does not hit, so by your argument you should also adjust radius by kAccuracy.